src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelectorAll('thead tr');
+const tbody = document.querySelectorAll('tbody tr');
+const tfoot = document.querySelectorAll('tfoot tr');
+
+const columnIndex = 1;
+
+thead.forEach((tr) => {
+  const th = tr.children[columnIndex];
+  const clonedTh = th.cloneNode(true);
+
+  tr.insertBefore(clonedTh, tr.children[columnIndex + 3]);
+});
+
+tbody.forEach((tr) => {
+  const th = tr.children[columnIndex];
+  const clonedTh = th.cloneNode(true);
+
+  tr.insertBefore(clonedTh, tr.children[columnIndex + 3]);
```

Comment:
Similar to the thead section, ensure that the tbody rows have enough columns to accommodate the insertion at `columnIndex + 3`. This will prevent potential errors if the row has fewer columns.